PopchukNataliya/HW6/Task_2.py

```diff
@@ -0,0 +1,10 @@
+# Script that checks the login that the user enters
+
```

Comment:
good job
look at this code
user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)

Reply:
Thank you very much for your valuable advice! I'll definitely take your code into consideration.